pkgs/servers/sql/postgresql/ext/timescaledb.nix

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
On the line above this, can you correct the comment to use `settings` instead of `extraConfig`?

Reply:
Done. And also included a comment that `pkgs`, which is in this case some variant of `postgresqlPackages` has to be taken from the correct `postgresql` package so the versions match.

Reply:
With this sequence of commands, it did run for me:
```
git fetch upstream pull/212467/head:pr_update_timescaledb_2.9.2
git switch pr_update_timescaledb_2.9.2
git apply - <<EOF                                                                              
∙ index 7e0a82c3882..7cb05fa18a1 100644                                                                            
--- a/nixos/tests/postgresql.nix                                                                                   
+++ b/nixos/tests/postgresql.nix                                                                                   
@@ -32,6 +32,10 @@ let                                                                                             
         services.postgresql = {                                                                                   
           enable = true;                                                                                          
           package = postgresql-package;                                                                                                                                                                                                
+          extraPlugins = with postgresql-package.pkgs; [                                                          
+            timescaledb                                                                                                                                                                                                                
+          ];                                                                                                              
+          settings.shared_preload_libraries = "timescaledb";                                                      
         };                                                                                                                 
                                                                                                                   
         services.postgresqlBackup = {                                                                             
                                                                                                                    
EOF
nix build --rebuild -L -f nixos/tests/postgresql.nix postgresql_13
```
pay attention how `timescaledb` is taken from `postgresql-package.pkgs`.

Reply:
The toplevel `timescaledb` attribute is meant to be used with the toplevel `postgresql` attribute, which is at the time of this pr `postgresql_14`.

Reply:
Ah excellent, sorry for the oversight-- works well now. Another formatting nit though; `postgresql_package` should _realllly_ be `postgresqlPackage`, or maybe even `postgresPackage`. 

NixOS (and most of Nixpkgs) tend to be camelCase everywhere unless it's something where we're explicitly translating into another format, like [RFC42](https://github.com/NixOS/rfcs/pull/42) `settings`. (Though, it seems the Postgres test is full of violations)